### PR TITLE
Fix osx install doc.

### PR DESCRIPTION
**Pull request by TimZaman**
- [x] Remove the brew update entry when installed with binary.
- [x] Add notion of the `.profile` file instead of `.bashrc` (though neither necessarily exists)

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

File: site/docs/install-os-x.md [hunk 1]
```diff
@@ -104,15 +110,12 @@ this directory to your default paths, as follows:
...
 ```bash
 bazel version
 ```
 
-Once installed, you can upgrade to a newer version of Bazel with:
-
-```bash
-brew upgrade bazel
-```
+When bazel is installed using the binary installer, updating bazel is only
```
Comment (spomorski) [inline site/docs/install-os-x.md]: Nit: "Bazel" (capitalized)

Comment (spomorski) [inline site/docs/install-os-x.md]: Nit: "Bazel" (capitalized)

**Comment by katre** [comment]
Waiting for @spomorski's comment to be resolved before I merge this internally.

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by TimZaman** [comment]
Aye sorry @katre ! Amended as requested.

**Pull request closed without merging**